### PR TITLE
Fix #117

### LINKED ISSUE #117: Add PID enabled to ViewPID display
We are able to enable/disable the PID, and the flag is used when computing the on-time for the CO2 bubbler, but we don't have a UI to view the state. We do have three lines that show the Kp, Ki, and Kd values. This would be a good place to add that view.

### PR DESCRIPTION
**Pull request by Paydenb**
Fix problem#117:
Now there is a UI that displays if PID is on or off

### CODE REVIEW

File: src/UIState/SeePIDConstants.h [hunk 1]
```diff
@@ -23,7 +23,8 @@ class SeePIDConstants : public UIState {
   void loadKp(uint16_t line);
   void loadKi(uint16_t line);
   void loadKd(uint16_t line);
-  void loadSlope(uint16_t line);
+  void loadPID(uint16_t line);
+//   void loadSlope(uint16_t line);
```
Comment (jgfoster) [inline src/UIState/SeePIDConstants.h:27]: If we find we don't need this file (things work with it commented out), let's remove the line.

**Review by jgfoster — CHANGES_REQUESTED** [review]
Delete one line...

File: src/UIState/SeePIDConstants.h (hunk 1 above)
Comment (jgfoster) [reply src/UIState/SeePIDConstants.h:27]: Note also that this line fails the formatting test, so when it is removed we should have passing tests!

**Review by jgfoster — APPROVED** [review]

**Review by prestoncarman — APPROVED** [review]
LGTM

**Comment by prestoncarman** [comment]
@all-contributors please add @Paydenb for code

**Pull request merged**